sys/fs/fuse/fuse_vnops.c

```diff
@@ -617,7 +618,7 @@ fuse_vnop_allocate(struct vop_allocate_args *ap)
 		return (EROFS);
 
 	if (fsess_not_impl(mp, FUSE_FALLOCATE))
-		return (EINVAL);
+		return (EXTERROR(EINVAL, "This daemon does not implement ALLOCATE"));
```

Comment:
I would be specific.  I find this less confusing.
```suggestion
		return (EXTERROR(EINVAL, "This daemon does not implement FUSE_FALLOCATE"));
```